dashboard/db/migrate/20241024171901_created_submitted_projects.rb

```diff
@@ -0,0 +1,13 @@
+class CreatedSubmittedProjects < ActiveRecord::Migration[6.1]
+  def change
+    # use utf8mb4 to support emojis
+    create_table :submitted_projects, charset: 'utf8mb4', collation: 'utf8mb4_bin' do |t|
+      t.integer :project_id
+      t.string :description
+      t.datetime :declined_at
+      t.timestamps
+    end
+
+    add_index :submitted_projects, :project_id, unique: true
```

Comment:
If we're typically going to be querying for all the non-declined projects on this table, might we want 
```
add_index :submitted_projects, :declined_at
```
?

It probably doesn't matter if we expect the table to be small, and we can always add one later. @fisher-alice @bencodeorg thoughts?

Reply:
Ah, whoops, just saw Ben's comment! 😄